Tasks/Aleksiutovich/Task6/Task_6_5.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+import random
+from functools import lru_cache
+
+# Write a decorator function that caches the return value of
+# a function with a dictionary.
+# If the function is called again with the same arguments,
+# return the cached value instead of computing it again.
+
+cache = {}
+
+
+def cache_decorator(func):
+    global cache

```

Comment:
it's a good decision to place the cache outside of the decorator, but the global is redundant since the dict is mutable already